projects/bad_example_mixed_sanitizers/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Copyright 2017 Google Inc.
```

Comment:
That's a good suggestion, I'll move all defect targets into a single project. I believe that it might be useful to land it, but not to report post build failures. We should report lack of proper failures for that project.